pkgs/tools/misc/mmctl/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, pkgs, fetchFromGitHub, buildGoModule }:
+buildGoModule rec {
+  pname = "mmctl";
+  version = "6.2.1";
+
+  src = fetchFromGitHub {
+    owner = "mattermost";
+    repo = "mmctl";
+    rev = "v${version}";
+    sha256 = "sha256-DhkBiczQ+4iwoufHMuy6Fn3a4q7gvkKGXTfLcUCCKis=";
+  };
+
+  vendorSha256 = null;
+
+  testPhase = "make test";
```

I don't see any other mentions of `testPhase` in nixpkgs.
```suggestion
  checkPhase = "make test";
```

---

Yes indeed, fixed! It did test though.